wallets/leapextension.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "../wallet.schema.json",
+  "wallet_name": "leapextension",
+  "pretty_name": "Leap Cosmos Wallet",
+  "website": "https://www.leapwallet.io/",
+  "git_repo": "",
+  "supported_chains": [
+    "cosmoshub",
+    "juno",
+    "osmosis",
+    "secretnetwork",
+    "agoric",
+    "akash",
+    "assetmantle",
+    "axelar",
+    "bitsong",
+    "cheqd",
+    "chihuahua",
+    "comdex",
+    "desmos",
+    "emoney",
+    "evmos",
+    "fetchhub",
+    "gravitybridge",
+    "injective",
+    "irisnet",
+    "jackal",
+    "kujira",
+    "likecoin",
+    "mars",
+    "persistence",
+    "atlantic",
```

Comment:
atlantic not registered to chain registry

Reply:
atlantic is registered on the [testnet registry](https://github.com/cosmos/chain-registry/tree/master/testnets/atlantic). Let me know if this list is supposed to include only mainnet chains, I'll update the pr accordingly.

Reply:
You are correct, 'atlantic' is registered to the chain registry as a testnet. At the moment, it is undetermined how exactly we'll handle testnets in the Wallet Registry. I'm inclined to temporarily allow this to be included until the decision is made.